### PR TITLE
Improve version bump around Az

### PR DESCRIPTION
**Pull request by dingmeng-xue**
## Description

Improvements in RunVersionController.ps1

- [x] * Check new added modules from the psd1 in local module. It can help to add new module to release notes. The list of modules need to be sorted because new module is appended at the end but it should be shown up at the correct position.
- [x] * Bump version as minor at least if Az just includes new modules but others have no any change or just patch release.
- [x] * Check release notes cannot contains non-ascii characters which doesn't allow by Nuget package.
- [x] * Show warning message when release notes contains more than 9000 characters. Nuget package allows 10000 characters at 

most.

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: tools/RunVersionController.ps1 [hunk 1]
```diff
@@ -290,6 +291,21 @@ switch ($PSCmdlet.ParameterSetName)
...
+        {
+            $galleryDependency = $galleryAz.Dependencies | where { $_.Name -eq $requiredModule.Name }
+            if ($galleryDependency -eq $null)
+            {
+                Write-Host "New module $($requiredModule.Name) is found in local Az" -ForegroundColor Yellow
+                $updatedModules += $requiredModule.Name
+                if ($versionBump -eq [PSVersion]::NONE -or $versionBump -eq [PSVersion]::PATCH)
+                {
+                    $versionBump = [PSVersion]::MINOR
+                    Write-Host "Adjust $versionBump version bump" -ForegroundColor Yellow
```
Comment (BethanyZhou) [inline tools/RunVersionController.ps1:304]: It's suggested to describe in more details, such as "Adjust $versionBump version bump for Az"

Comment (dingmeng-xue) [reply tools/RunVersionController.ps1]: Updated.

**Review by BethanyZhou — APPROVED** [review]

**Pull request closed without merging**